lib/v3/parser.js

```diff
@@ -227,53 +230,75 @@ class Parser extends EventEmitter {
         this.emit('component', item);
     }
 
+    emitGlobalComment(comment) {
+        if (utils.isTopLevelComment(comment)) {
+            if (this.features.includes('description')) this.emit('description', comment.description);
+            if (this.features.includes('keywords')) this.emit('keywords', comment.keywords);
+        }
+    }
+
     parseBodyRecursively(rootNode, parseContext, level) {
         const nodes = rootNode.body
             ? rootNode.body
             : (rootNode.length > 0 ? rootNode : [rootNode]);
 
-        nodes.forEach(node => {
+        nodes.forEach((node, index) => {
+            if (index === 0) {
+                const firstComment = utils.getComment(node, { useTrailing: false, useFirst: true });
+
+                this.emitGlobalComment(firstComment);
```

Comment:
Not sure complete what happens when `null` are passed to this method, this case is OK?

Reply:
Yes passing null would break it (as it would in previous setup as well), but I did add a default value of empty object so it works when passed `undefined`.

Reply:
you ment to `getComment`right? or did you mean to `emitGlobalComment`?

Reply:
I mean case when `utils.getComment` returns an empty value. So I think we need add a check here before calling `emitGlobalComment`

Reply:
Does it ever return undefined or null? Either way I'll add a check, better be safe 😉